addons/vitals/stringtable.xml

```diff
@@ -7,18 +7,21 @@
             <Spanish>Activar Médico Simple para IA</Spanish>
             <Japanese>AI用単純化医療を有効化</Japanese>
             <Italian>Attiva AI Simple Medical</Italian>
+            <German>Vereinfachte med. Simulation für KI</German>
             <French>Simplifier la médecine pour l'IA.</French>
         </Key>
         <Key ID="STR_KAT_Vitals_SIMPLEMED_ENABLE_DESC">
             <English>Apply simplified vital calculations to the AI unit. Which can be useful for saving FPS.</English>
             <Japanese>単純化されたバイタル計算をAIユニットに適用します。 これは負荷の軽減に役立つ可能性があります。</Japanese>
             <Italian>Applica calcoli vitali semplificati all'unità AI. Che può essere utile per salvare FPS.</Italian>
+            <German>Wenden Sie vereinfachte Vitalberechnungen auf die KI-Einheit an. Was zum Sparen von FPS nützlich sein kann.</German>
```

Comment:
"Wendet vereinfachte Vitalberechnungen auf KI-Einheit an. Was für eine bessere Leistung nützlich sein kann." would be better